src/main/java/org/commcare/cases/instance/CaseChildElement.java

```diff
@@ -321,7 +146,7 @@ private void setCaseProperties(Case c, TreeElement cacheBuilder) {
                 continue;
             }
 
-            TreeElement scratch = new TreeElement(parent.intern(key));
```

Comment:
I think I was probably in destructive mood and decided to rip out the unused string interner; going against a previous discussion we'd had about potentially trying to hook up the string interner again. I'll look into adding it back :)